src/main/java/org/opentripplanner/middleware/utils/Coordinates.java

```diff
@@ -6,14 +6,14 @@
  * Helper class to contain lat/lon values.
  */
 public class Coordinates {
-    public double lat;
-    public double lon;
+    public Double lon;
+    public Double lat;
 
     /** Required for JSON serialization */
     public Coordinates() {
     }
 
-    public Coordinates(double lat, double lon) {
+    public Coordinates(Double lat, Double lon) {
```

Comment:
Instead of allowing nulls, would it be better to not provide/persist an instance of `Coordinates` in both `AnonymizedTripRequest` and `AnonymizedLeg` the trip if both `lat` and `lon` are null?